### PR TITLE
23572 update dissolution letters

### PR DESCRIPTION
**Pull request by kzdev420**
*Issue #:* /bcgov/entity#23572

*Description of changes:*
[BC-Corp.pdf](https://github.com/user-attachments/files/17317767/BC-Corp.pdf)
[PRTA.pdf](https://github.com/user-attachments/files/17317776/PRTA.pdf)
[XPRO.pdf](https://github.com/user-attachments/files/17317778/XPRO.pdf)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/report-templates/noticeOfDissolutionCommencement.html [hunk 1]
```diff
@@ -75,17 +80,15 @@
...
-          <div class="mt-5">If your company is dissolved under section 422(1)(a) of the Act, section 347 of the Act states the liability of each director, officer, shareholder and liquidator of a company that is dissolved continues and may be enforced as if the company had not been dissolved.</div>
+          <div class="mt-5">If your company is dissolved under section 422(1)(f) of the Act, section 347 of the Act states the liability of each director, officer, shareholder and liquidator of a company that is dissolved continues and may be enforced as if the company had not been dissolved.</div>
           <div class="mt-5">If you have filed the outstanding PRTA, no further action is required.</div>
           <div class="mt-5">If you need help with setting up an account or managing a business, please visit our Resources and Help page at <a href="https://bcreg.ca/resources" class="break-url">bcreg.ca/resources</a></div>
```
Comment (eason-pan-bc) [inline legal-api/report-templates/noticeOfDissolutionCommencement.html:93]: I think, we may miss a `.` at the end of the sentence, although it's also missed in the doc files in the ticket. 
```suggestion
          <div class="mt-5">If you need help with setting up an account or managing a business, please visit our Resources and Help page at <a href="https://bcreg.ca/resources" class="break-url">bcreg.ca/resources</a>.</div>
          
```

File: jobs/furnishings/report-templates/noticeOfDissolutionCommencement.html [hunk 2]
```diff
@@ -32,23 +32,28 @@
...
+            <div class="mt-5">Under section 422 of the Business Corporation Act (the Act), this letter is to notify you that your extraprovincial company has for two years failed to file the annual reports required under section 380 of the Act. An extraprovincial company must annually, within two months after each anniversary of the date of its registration, file an annual report with the registrar.</div>
+            <div class="mt-5">If within one month after the date of this letter, the company fails to file the outstanding annual reports, a notice may be published on the BC Laws website <a href="https://www.bclaws.gov.bc.ca/" class="break-url">www.bclaws.ca</a>. This notice will state that, at any time after the expiration of one month after the date of publication of the notice, the company’s registration may be cancelled, unless cause is shown to the contrary; The Registrar satisfied the failure has been or is being remedied; or a copy of the entered court order to the contrary has been filed.</div>
```
Comment (AimeeGao) [inline jobs/furnishings/report-templates/noticeOfDissolutionCommencement.html:49]: image
I noticed that the letter 't' in 'D1 Dissolution Letter - XPros - policy revision' is in lowercase. Perhaps we could follow the template's style for consistency.

Comment (kzdev420) [reply jobs/furnishings/report-templates/noticeOfDissolutionCommencement.html:49]: Yes, I saw it. But, they are uppercase in the other letters ( BC Corp and PRTA ). So, I made it like that.
@dylan644 can you confirm it?

Comment (dylan644) [reply jobs/furnishings/report-templates/noticeOfDissolutionCommencement.html:49]: well spotted - let's go with consistency please

**Review by AimeeGao — APPROVED** [review]
LGTM🚀

**Review by eason-pan-bc — APPROVED** [review]
LGTM 👍

**Pull request merged**